### PR TITLE
feat: support yurtadm config command

### LINKED ISSUE #1536: [feature request] Support yurtadm config command
**What would you like to be added**:
Support the `yurtadm config` command to provide the user with the default `JoinConfiguration`.

**Why is this needed**:
As discussed in #1448, we need to give yurtadm join a `--config` option. this requires that we can somehow give the user a default config template, so this requirement wants to give the user a default config template using the new `yurtadm config` command.

The implementation of `yurtadm config` can be refer to `kubeadm config`.

**others**
/kind feature

### PR DESCRIPTION
**Pull request by YTGhost**
#### What type of PR is this?
/kind feature

#### Which issue(s) this PR fixes:

Fixes #1536 

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by YTGhost** [comment]
@rambohe-ch PTAL

File: pkg/yurtadm/cmd/config/config.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	KubeadmJoinDiscoveryFilePath := filepath.Join(constants.KubeletWorkdir, constants.KubeadmJoinDiscoveryFileName)
+	ignoreErrors := sets.String{}
+	ctx := map[string]interface{}{
+		"kubeConfigPath":         KubeadmJoinDiscoveryFilePath,
+		"tlsBootstrapToken":      constants.PlaceholderToken,
+		"ignorePreflightErrors":  ignoreErrors.List(),
+		"podInfraContainerImage": constants.PauseImagePath,
+		"nodeLabels":             fmt.Sprintf("%s=true", projectinfo.GetEdgeWorkerLabelKey()),
+		"criSocket":              constants.DefaultDockerCRISocket,
+		"name":                   constants.EdgeNode,
```
Comment (rambohe-ch) [inline pkg/yurtadm/cmd/config/config.go:100]: we should specify field `name` with node name instead of node type.

Comment (YTGhost) [reply pkg/yurtadm/cmd/config/config.go]: Fixed

**Comment by YTGhost** [comment]
@rambohe-ch PTAL

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**